Intacct.SDK.Tests/Functions/AccountsReceivable/ArAdvanceCreateTest.cs

```diff
@@ -0,0 +1,162 @@
+﻿/*
+ * Copyright 2020 Sage Intacct, Inc.
```

Comment:
see if you can update these to `2022` (here and in other new files as part of this PR) ; you don't have to update old ones